prime-router/metadata/schemas/iPatientCare/iPatientCare-covid-19.schema

```diff
@@ -0,0 +1,323 @@
+    - name: testing_lab_name
+      csvFields: [{name: Facility}]
+
+    - name: testing_lab_street
+      csvFields: [{name: Fac_Addr1}]
+
+    - name: testing_lab_city
+      csvFields: [{name: Fac_City}]
+
+    - name: testing_lab_state
+      csvFields: [{name: Fac_State}]
+
+    - name: testing_lab_zip_code
+      csvFields: [{name: Fac_Zip}]
+
+    - name: testing_lab_phone_number
+      csvFields: [{name: Fac_Phone}]
+
+
+#misc
+    - name: reporting_facility_name
+      csvFields: [{name: Facility}]
+
+    - name: filler_name
+      csvFields: [{name: Facility}]
+
+    - name: patient_id_assigner
+      csvFields: [{name: Facility}]
+
+
+#CLIA
+    - name: testing_lab_clia
+      csvFields: [{name: CLIA No}]
+
+    - name: testing_lab_id
+      csvFields: [{name: CLIA No}]
+
+    - name: reporting_facility_clia
+      csvFields: [{name: CLIA No}]
+
+    - name: filler_clia
+      csvFields: [{name: CLIA No}]
+
+
+
+#specimen/accession fields
+    - name: filler_order_id
+      csvFields: [{name: Accession_no}]
+
+
+
+#AOE fields
+    - name: pregnant
+      csvFields: [{name: Pregnant, format: $alt}]
+      altValues:
+        - code: 77386006
+          display: Y
+        - code: 77386006
+          display: YES
+        - code: 77386006
+          display: Pregnant
+        - code: 77386006
+          display: Currently Pregnant
+        - code: 60001007
+          display: N
+        - code: 60001007
+          display: NO
+        - code: 60001007
+          display: Not Pregnant
+        - code: 60001007
+          display: Not Currently Pregnant
+        - code: 261665006
+          display: U
+        - code: 261665006
+          display: UNK
+        - code: 261665006
+          display: Unknown
+
+    - name: hospitalized
+      default: N
+
+    - name: icu
+      default: N
+
+
+#this lookup is in the covid-19 schema, but doesn't appear to execute??
+    - name: equipment_model_name
+      table: LIVD-SARS-CoV-2-2021-04-28
+      tableColumn: Model
+      mapper: livdLookup()
+
+
+#unused fields - here to suppress warnings.
+    - name: Lab name
+      type: TEXT
+      csvFields: [{name: Lab name}]
+
+    - name: SSN
+      type: TEXT
+      csvFields: [{name: SSN}]
+
+    - name: LOINC
+      type: TEXT
+      csvFields: [{name: LOINC}]
+
+    - name: ResultUnits
+      type: TEXT
+      csvFields: [{name: ResultUnits}]
+
+    - name: Reference Range
+      type: TEXT
+      csvFields: [{name: Reference Range}]
+
+    - name: PhyAddress1
+      type: TEXT
+      csvFields: [{name: PhyAddress1}]
+
+    - name: PhysCity
+      type: TEXT
+      csvFields: [{name: PhysCity}]
+
+    - name: PhysST
+      type: TEXT
+      csvFields: [{name: PhysST}]
+
+    - name: PhysZip
+      type: TEXT
+      csvFields: [{name: PhysZip}]
+
+    - name: PhysPhone
+      type: TEXT
+      csvFields: [{name: PhysPhone}]
+
+    - name: Comments
+      type: TEXT
+      csvFields: [{name: Comments}]
+
+    - name: Specimen_Type_Ignore
```

Comment:
Why does this one have _Ignore tacked on the end, and the others don't?

Reply:
I'll comment/document this one better.  There's a couple things going on here.  Their extract uses the same column name "Specimen_Type" as RS does internally so I had to name it something different.  It may be better just to append ignore on all of them, though.

Secondly, the sample data we have contains either an empty field or "NULL", so we can't use this for the specimen type anyway.  Hoping we can find out more with larger volumes and/or a different iPatientCare client.

Reply:
I added the "_ignore" suffix to the others and added documentation that now shows up in the generated docs.